examples/ui/ui.rs

```diff
@@ -95,12 +96,7 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
                             },
                         )
                         .with_style(Style {
-                            size: Size::new(Val::Undefined, Val::Px(25.)),
-                            margin: UiRect {
-                                left: Val::Auto,
-                                right: Val::Auto,
-                                ..default()
-                            },
```

Comment:
Auto margins are *not* the default (or should not be). For margins, "auto" means "take up available space". The default margin is zero.